quadrat/inc/patterns/search-results.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Search results
+ *
+ * @package Quadrat
+ */
+
+return array(
+	'title'       => __( 'Search results', 'quadrat' ),
+	'categories'  => array( 'quadrat' ),
+	'content'     => '<!-- wp:group {"layout":{"inherit":true}, "className":"page-content"} -->
+		<div class="wp-block-group page-content">
+
+		<!-- wp:heading -->
+		<h2>' . __('Results') . ':</h2>
```

Comment:
This is tangent to the question of the [Pattern block](https://github.com/WordPress/gutenberg/pull/33217), but the colon should be a part of the string, allowing translators to localise accordingly:

```suggestion
		<h2>' . __('Results:') . '</h2>
```

This is a boon for languages that follow different punctuation conventions (e.g. alternatives to the colon, or spacing prescriptions around the colon), but also — I expect, without having tested — a way to avoid flow bugs manifesting in RTL languages.

Reply:
Thanks, TIL :)